plugins/experimental/traffic_dump/traffic_dump.cc

```diff
@@ -304,7 +304,7 @@ session_aio_handler(TSCont contp, TSEvent event, void *edata)
         ts::file::file_status st = ts::file::status(ssnData->log_name, ec);
         if (!ec) {
           disk_usage += ts::file::file_size(st);
-          TSDebug(PLUGIN_NAME, "Finish a session with log file of %" PRId64 "bytes", ts::file::file_size(st));
+          TSDebug(PLUGIN_NAME, "Finish a session with log file of %ju bytes", ts::file::file_size(st));
```

Comment:
No, that's worse, less portable. The better fix is to change "PRId64" to  "PRIu64".

The best fix, of course, would be to use `BufferWriter::print`, which knows the type of the integer and does the correct thing automatically.

Reply:
So, wrapping up the discussions on IRC, lets use PRIuMAX to be compatible going forward.